boards/riscv/esp32c3_devkitm/esp32c3_devkitm.yaml

```diff
@@ -3,4 +3,4 @@ name: ESP32-C3
 type: mcu
 arch: riscv
 toolchain:
-  - espressif
+  - zephyr
```

Comment:
Any reason to remove `espressif` here? Does Espressif's toolchain no longer support ESP32C3?

Reply:
Currently esp32c3 uses zephyr toolchain for building its application code, but the bootloader was being compiled using the IDF toolchain.

The idea is to use only the Zephyr toolchain for both and extend this to other espressif SoC.

This would remove the dependency of dealing with IDF toolchain manual setting when preparing the build setup for ESP32 and its variants.

Reply:
Fair enough.